### DISCUSSION TITLE
Is it possible to have 1.0 and 1.1? I wondered if I could rename old folder and then install 1.1?

### DISCUSSION THREAD

**Discussion by creeduk** [Mikubill/sd-webui-controlnet#1060] [Q&A]
I am backing up 1.0 as I have an ongoing project which is not complete although I keep having issues with canny and or pose accuracy so I am hoping 1.1 will help but need to go back quickly if not.

Also has anybody documented the difference between the 1.1. canny, sketch etc. and the 1.0 ones? Curious how much improvement the older methods have while looking forward to the new ones.

**Comment by creeduk** [comment]
It seemed to load but some scribble errors, all models  from both folders available. I had issues of AttributeError: module 'torch' has no attribute 'concatenate' so I cleaned up and found that is an issue with setting "ControlNet is more important" option with any alt models t2i or even coadpater models

**Comment by Joeyd348039** [comment]
Is there a link to the first version of Controlnet that had reference_only added in? Think it was the 1.1.153?
